### PR TITLE
Fix simplified page being slow due to very large text

### PR DESCRIPTION
**Pull request by frozenhelium**
- Limit the amount of characters displayed in simplified text
- Add 'Show more' button at the end of simplified text
- Fix unexpected jumps when adding an entry form simplified text
- Add tooltip for ellipsized text in SubNavbar
- Improve styling of edit excerpt popup
- Check commonAncestorContainer before accessing bounding client rect
- Sort entries from server on ascending order of date created
- Newly added entries are added to the bottom
- Use ref instead of state for storing staleIdentifiers and deleteIdentifiers

### CODE REVIEW

File: app/views/Project/EntryEdit/LeftPane/SimplifiedTextView/index.tsx [hunk 1]
```diff
@@ -54,9 +59,32 @@ function SimplifiedTextView(props: Props) {
...
+        // Note: we need to preserve scroll due to
+        // unexpected jumps on entry addition / removal
+        const scrollTop = containerRef.current?.scrollTop;
+        if (isDefined(scrollTop)) {
+            window.setTimeout(() => {
+                if (containerRef.current) {
+                    containerRef.current.scrollTop = scrollTop;
+                }
+            }, 0);
+        }
```
Comment (tnagorra) [inline app/views/Project/EntryEdit/LeftPane/SimplifiedTextView/index.tsx]: We shouild put this in useEffect

File: app/views/Project/EntryEdit/LeftPane/SimplifiedTextView/styles.css [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
     padding: var(--padding);
-    text-align: justify;
+
     white-space: pre-wrap;
     word-break: break-word;
 
+    * {
+        white-space: pre-wrap;
+        word-break: break-word;
+    }
```
Comment (tnagorra) [inline app/views/Project/EntryEdit/LeftPane/SimplifiedTextView/styles.css]: Do we need to change this?

**Review by tnagorra — DISMISSED** [review]

File: app/views/Project/EntryEdit/LeftPane/SimplifiedTextView/styles.css (hunk 2 above)
Comment (frozenhelium) [reply app/views/Project/EntryEdit/LeftPane/SimplifiedTextView/styles.css]: Removed

**Review by AdityaKhatri — APPROVED** [review]

**Review by samshara — APPROVED** [review]

**Comment by samshara** [comment]
DeepScan issues need to be fixed @frozenhelium

**Pull request merged**